Assets/Scripts/Controllers/MainMenuController.cs

```diff
@@ -62,10 +74,14 @@ public void Start()
 
         // Dev Console
         GameObject devConsole = (GameObject)Instantiate(Resources.Load("UI/Console/DevConsole"));
-        devConsole.name = "DevConsole";
-        devConsole.transform.SetParent(canvas.transform, false);
-        devConsole.transform.SetAsLastSibling();
-        devConsole.SetActive(false);
-        DeveloperConsole.DevConsole.Close();
+
+        if (devConsole != null)
```

Comment:
I'm a little bit confused, why would DevConsole be null? You just Instantiated it.